src/psygnal/_group.py

```diff
@@ -40,95 +42,45 @@ class EmissionInfo(NamedTuple):
     args: tuple[Any, ...]
 
 
-@mypyc_attr(allow_interpreted_subclasses=True)
-class SignalGroup(SignalInstance):
-    """`SignalGroup` that enables connecting to all `SignalInstances`.
+class SignalRelay(SignalInstance):
```

Comment:
Create variant that allow to connect to not deprecated signals?

Reply:
I'm not following what you mean by allow to connect to not deprecated signals

Reply:
do you mean a variant that doesn't deprecate direct access to all of the `SignalInstance` names?
If so, I would ask why?  Do you think it's actually a better design?  Or don't want to have to deal with the deprecation? (fwiw, I tested against napari and magicgui, napari is fine, and magicgui is fine after https://github.com/pyapp-kit/magicgui/pull/629).  
I'd rather just decide whether it's deprecated or not, rather than have a variant that is deprecated and one that isn't... but let me know if I misunderstood you there